### ISSUE TITLE
Unable to launch WLJS

### ISSUE THREAD

**Issue by xiaoxuan-yu** [JerryI/wolfram-js-frontend#161]
**Describe the bug**
Not able to launch wljs-notebook.

**To Reproduce**
Install the latsest wljs-notebook with deb/appimage, launch the software.

**Expected behavior**
The software do not launch. When trying to launch by terminal, the error message below is printed as the output
```
Mica-Electron only works on Windows!
Mica-Electron: Disabled
aaahhh...
exiting the server...
(node:1047594) UnhandledPromiseRejectionWarning: Error: Failed to load image from path '/opt/WLJS Notebook/resources/app/Electron/build/256x256_new.png'
    at /opt/WLJS Notebook/resources/app/Electron/main.js:1318:16
    at emitUnhandledRejectionWarning (node:internal/process/promises:200:15)
    at processPromiseRejections (node:internal/process/promises:296:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
(node:1047594) Error: Failed to load image from path '/opt/WLJS Notebook/resources/app/Electron/build/256x256_new.png'
    at /opt/WLJS Notebook/resources/app/Electron/main.js:1318:16
```

**Screenshots**
![image](https://github.com/JerryI/wolfram-js-frontend/assets/55308316/b722b104-beb0-4d70-b083-46cb82d26569)

**Desktop (please complete the following information):**
 - OS: KUbuntu 22.04
 - Browser [e.g. chrome, safari] or Desktop App: Desktop App

**Comment by JerryI** [comment]
Could you, please, try this one?

[link](https://dropmefiles.com/2sp6g)

**Comment by JerryI** [comment]
We have recently updated the binaries, and probably missed some files in the bundle while packing it for Linux

**Comment by JerryI** [comment]
We have updated the releases page [link](https://github.com/JerryI/wolfram-js-frontend/releases/tag/2.3.5).
I checked on Debian 11, now it works

![IMG_DCEE277A1775-1](https://github.com/JerryI/wolfram-js-frontend/assets/4111822/7b3c4ed7-4fe7-488b-9ed1-de7e6a50ee07)

**Comment by xiaoxuan-yu** [comment]
The problem that image failed to load seems solved, no error message printed in the terminal. However, the ui seems not able to load correctly.
![image](https://github.com/JerryI/wolfram-js-frontend/assets/55308316/ef07b324-96cc-495e-83a4-ac1599b940d8)
The output is pasted as below
```
Mica-Electron only works on Windows!
Mica-Electron: Disabled
aaahhh...
exiting the server...
shortcut required
shortcut required
shortcut required
shortcut required
shortcut required
shortcut required
shortcut required
shortcut required
Electron >> starting powersafe blocker
```
I fully understand that it is an independent problem, maybe I need yo open a new issue?

**Comment by JerryI** [comment]
Hmm… Another question. Do you use “_minimal” or normal installation file?

**Comment by xiaoxuan-yu** [comment]
the normal one

**Comment by JerryI** [comment]
We ve found a Kubuntu machine with @KirillBelovTest and will try to test everything there.

**Comment by JerryI** [comment]
We found out the solution.
```
sudo apt install zenity
```

Probably this is bad, that we rely on this package, which is missing on some linux distros. In the next release soon we will get rid of this dependency.

**Comment by JerryI** [comment]
Must be fixed in #165
